### PR TITLE
Checkin of April meeting minutes

### PR DESCRIPTION
**Pull request by SimonDarksideJ**

### CODE REVIEW

**Review by dellis1972 — DISMISSED** [review]

File: website/content/blog/2024-04-10-meeting.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+## Bounty tracker
+
+All bounties are still tracked primarily from the [Bounties](https://github.com/MonoGame/MonoGame/issues/8120) GitHub issue.  Each bounty has its own issue to keep communication open (subject to NDA requirements), including:
+
+- [A shared base for public and console repos](https://github.com/MonoGame/MonoGame/issues/8242) - **Still open**
+
+Upcoming bounties:
+
+- A more modern getting started (installation process, content pipeline and 1st simple game)
+- A cross platform MonoGame setup,  including the content pipeline
```
Comment (harry-cpp) [inline website/content/blog/2024-04-10-meeting.md:49]: there is an extra space

File: website/content/blog/2024-04-10-meeting.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+This will ensure a fair and speedy process and reduce any "red tape" for getting work picked up quickly, while still ensuring all interested applicants voices are heard.
+
+- Metal implementation is fine, so long as it follows the new native api.
+
+## Bounty tracker
+
+All bounties are still tracked primarily from the [Bounties](https://github.com/MonoGame/MonoGame/issues/8120) GitHub issue.  Each bounty has its own issue to keep communication open (subject to NDA requirements), including:
+
+- [A shared base for public and console repos](https://github.com/MonoGame/MonoGame/issues/8242) - **Still open**
```
Comment (harry-cpp) [inline website/content/blog/2024-04-10-meeting.md:44]: Its no longer open XD

File: website/content/blog/2024-04-10-meeting.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Upcoming bounties:
```
Comment (harry-cpp) [inline website/content/blog/2024-04-10-meeting.md:46]: If you are already mentioning the upcoming bounties, they are:
- MonoGame.Tool.X / MonoGame.Library.X for texture compression (waiting on ffmpeg bounty)
- Content Pipeline library hookup of all the Tool.X / Library.X repos (waiting on the above bounty)
- FAudio hookup (waiting on the native base bounty)

File: website/content/blog/2024-04-10-meeting.md (hunk 2 above)
Comment (SimonDarksideJ) [reply website/content/blog/2024-04-10-meeting.md:44]: My bad, left in the wrong option.

**Review by harry-cpp — APPROVED** [review]

**Pull request merged**